tsdb/test/labels_test.go

```diff
@@ -1,216 +0,0 @@
-// Copyright 2017 The Prometheus Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package test
-
-import (
-	"bytes"
-	"crypto/rand"
-	"testing"
-
-	"github.com/prometheus/prometheus/model/labels"
```

Comment:
Those benchmarks actually benchmark labels, so our code, no?

Reply:
OK wasn't true that Prometheus never calls `copy` on labels, but it is now I merged #11717 :-)
I guess the `labels` package could benchmark `labels.Copy()`,

Reply:
I moved the label-copying benchmark into the `labels` package.